server/utils/nunjucksSetup.ts

```diff
@@ -171,5 +171,15 @@ export function registerNunjucks(app?: express.Express): Environment {
     return staticPath(params)
   })
 
+  njkEnv.addFilter('formatListAsString', (list: { [key: string]: string }): string => {
+    if (list) {
+      return Object.keys(list)
```

Comment:
**suggestion:**  if you use `Object.values()` instead you don't need the mapping step